### PR TITLE
fix: ssd_512_resnet18_v1 settings

### PR DESCRIPTION
**Pull request by shesung**
Stride for stage3_activation1  in resnet18_v1 should be 16. 
Original steps will lead to nan in trainning.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-441-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-441/1/index.html 
Code coverage of this PR:  vs. Master:

File: gluoncv/model_zoo/ssd/ssd.py [hunk 1]
```diff
@@ -501,7 +501,7 @@ def ssd_512_resnet18_v1_voc(pretrained=False, pretrained_base=True, **kwargs):
                    filters=[512, 512, 256, 256],
                    sizes=[51.2, 102.4, 189.4, 276.4, 363.52, 450.6, 492],
                    ratios=[[1, 2, 0.5]] + [[1, 2, 0.5, 3, 1.0/3]] * 3 + [[1, 2, 0.5]] * 2,
-                   steps=[8, 16, 32, 64, 128, 256, 512],
+                   steps=[16, 32, 64, 128, 256, 512],
```
Comment (zhreshold) [inline gluoncv/model_zoo/ssd/ssd.py]: removing this will cause steps not matching sizes/ratios

Comment (shesung) [reply gluoncv/model_zoo/ssd/ssd.py]: There are 6 feature layers and 6 ratios. So, 6 steps will be ok.

**Comment by zhreshold** [comment]
Thanks @shesung, this is merged.

**Pull request merged**